modules/ROOT/pages/import/file-provision.adoc

```diff
@@ -2,23 +2,29 @@
 = File provision
 :description: This section describes how to provide files for import.
 
+You start by connecting your data source, which can be a SQL database or local flat files.
+
 In essence, you provide a set of files that contain the data to be imported and Import imports them into your instance.
 
-Import supports flat files, i.e. files that contain data in a tabular format where each row represents a record and each column represents a field in that record.
-The most common format for such files is CSV (comma-separated values), but Import also supports TSV (tab-separated values).
+Import supports SQL and flat files, i.e. files that contain data in a tabular format where each row represents a record and each column represents a field in that record.
+The most common format for flat files is CSV (comma-separated values), but Import also supports TSV (tab-separated values).
+
+When connecting to a remote data source, the files are provided for you from that database.
```

Comment:
I'd avoid calling these files. People will think of these as tables when coming from an RDBMS

Reply:
Makes sense, I'll rewrite.